ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/transaction/arjunacore/jca/TransactionImporter.java

```diff
@@ -60,6 +61,8 @@
 	 */
     public SubordinateTransaction importTransaction(Xid xid, int timeout) throws XAException;
 
+    public TransactionImportResult importForeignTransaction(Xid xid, int timeout) throws XAException;
```

Comment:
If that is the only change request I will push the update otherwise I will hold back until you have finished the review?